pkgs/development/python-modules/mistune/default.nix

```diff
@@ -4,8 +4,9 @@ self: rec {
     sha256 = "59a3429db53c50b5c6bcc8a07f8848cb00d7dc8bdb431a4ab41920d201d4756e";
   };
   mistune_2_0 = self.callPackage ./common.nix {
-    version = "2.0.0rc1";
-    sha256 = "1nd7iav1ixh9hlj4hxn6lmpava88d86ys8rqm30wgvr7gjlxnas5";
+    version = "2.0.0";
+    sha256 = "sha256-3nGPgtuMNzD4hdV2I3vcgZ3LuLwTQNs3jCe6iKuy13c=";
+    format = "pyproject";
   };
```

Comment:
Since this is a full release now we could drop mistune_0_8? Normally we do not want to have such a situation in pythonPackages at all.

Reply:
2.0 have some breaking changes, there are ~10 packages in nixpkgs that still use 0.8 and patches required to get it work with 2.0. I think it's better to handle it as separate PR.